templates/search/searchform.tpl

```diff
@@ -11,11 +11,11 @@
       <select name="type">
         <option value="Request">... requested username</option>
         <option value="email">... email address</option>
+				<option value="IP">... IP address</option>
         {if $currentUser->isAdmin() || $currentUser->isCheckuser()}
```

Comment:
... and while you're editing this file, let's remove the useless CIDR options and the duplicate IP option if you're a tool admin...

... which leaves the entire if block empty, and thus also unnecessary.